src/api/configureApi.ts

```diff
@@ -105,7 +105,6 @@ export function configureAndStartApi(useMetrics: boolean = true, portNumber: num
       makeMetricsApiMiddleware(
         {
           isNormalizePathEnabled: true,
-          discardUnmatched: false,
```

Comment:
I don't think having path normalization without discarding unmatched paths is useful, we still get all the spam URLs. If we want to disable this feature for some reason, should just flip the isNormalizePathEnabled switch.